kld/src/ldk/controller.rs

```diff
@@ -665,6 +670,11 @@ impl Controller {
             }
         };
         let channel_manager: Arc<ChannelManager> = Arc::new(channel_manager);
+        let liquidity_manager = Arc::new(LiquidityManager::new(
```

Comment:
I think you need to call wait_next_event or get_and_clear_pending_events from https://github.com/lightningdevkit/ldk-lsp-client/blob/main/src/transport/message_handler.rs#L113

Reply:
No, we do not need that.

The `wait_next_event` or `get_and_clear_pending_events` are getting the events and not LSPS0 itself so they should not be included in the PR. 

I think only in the following two scenarios we will need events.
- log every into the database
- do an LSPS query and wait for corresponding events

Furthermore, there is no event upstream currently.
https://github.com/lightningdevkit/ldk-lsp-client/blob/2fa04786d6750a650f66c807f717b30b2e7694e3/src/events.rs#L58

Reply:
OK then that's fine.

What about get_and_clear_pending_msg()? I am thinking about what would happen if a peer sends us an invalid message here https://github.com/lightningdevkit/ldk-lsp-client/blob/main/src/transport/message_handler.rs#L187

Reply:
I realised get_and_clear_pending_msg is called by LDK.